sdks/java/core/src/test/java/org/apache/beam/sdk/runners/dataflow/TestCountingSource.java

```diff
@@ -164,21 +168,20 @@ public CountingSourceReader(int startingPoint) {
 
     @Override
     public boolean start() {
-      return true;
+      return advance();
```

Comment:
This guy is on the way out - it should be replaced by CountingSource, so I'm not going to go to town here.